modules/3d/src/rgbd/odometry_functions.hpp

```diff
@@ -0,0 +1,213 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+
+#ifndef ODOMETRY_FUNCTIONS_HPP
+#define ODOMETRY_FUNCTIONS_HPP
+
+#include "../precomp.hpp"
```

Comment:
precomp.hpp is not allowed in .hpp files.

Reply:
done